tests/unit/mixins/FluxibleMixin.js

```diff
@@ -54,6 +127,10 @@ describe('StoreListenerMixin', function () {
                 statics: {
                     storeListeners: [MockStore]
                 },
+                childContextTypes: {
```

Comment:
React throws errors for public types on the context not declared in the `childContextTypes`.  This may be only for the top parent component.  Since the Mock context provides those additional public items they have to be declared or React pukes.

```
1) StoreListenerMixin getChildContext should propagate the context to child components:
     Error: Invariant Violation: ReactCompositeComponent.getChildContext(): key "dispatcher" is not defined in childContextTypes.
```